### PR TITLE
port control launch

### PR DESCRIPTION
**Pull request by TakaHoribe**
## How to test

```
 ros2 launch control_launch control.launch.xml 
```

### CODE REVIEW

**Review by esteve — CHANGES_REQUESTED** [review]

File: control_launch/package.xml [hunk 1]
```diff
@@ -4,19 +4,13 @@
   <version>0.1.0</version>
   <description>The control_launch package</description>
 
-  <!-- One maintainer tag required, multiple allowed, one person per tag -->
-  <!-- Example:  -->
-  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
   <maintainer email="horibe.takamasa@gmail.com">Takamasa Horibe</maintainer>
 
+  <license>Apache 2.0</license>
```
[the remark this answers is not in the document]
Comment (TakaHoribe) [reply control_launch/package.xml]: Thanks, fixed.

File: control_launch/config/mpc_follower/mpc_follower_param.yaml [hunk 2]
```diff
@@ -1,42 +1,42 @@
+/**:
```
[the remark this answers is not in the document]
Comment (TakaHoribe) [reply control_launch/config/mpc_follower/mpc_follower_param.yaml:1]: What do you mean? I ported all param files in this package.

[the remark this answers is not in the document]
Comment (esteve) [reply control_launch/config/mpc_follower/mpc_follower_param.yaml:1]: The params files are commented, they don't seem to be used anywhere.

[the remark this answers is not in the document]
Comment (TakaHoribe) [reply control_launch/config/mpc_follower/mpc_follower_param.yaml:1]: Are you talking about the free-namespace `/**:` ? All parameters are loaded through the `control.launch.xml` and applied to each node. You can launch and check them.

[the remark this answers is not in the document]
Comment (TakaHoribe) [reply control_launch/config/mpc_follower/mpc_follower_param.yaml:1]:
```
 ros2 launch control_launch control.launch.xml 
```

**Review by esteve — APPROVED** [review]

**Pull request merged**